docs/source/configuration_file.rst

```diff
@@ -794,6 +794,26 @@ Split Dataset
     }
 
 
+.. jsonschema::
+
+    {
+        "$schema": "http://json-schema.org/draft-04/schema#",
+        "title": "control_randomness",
+        "description": [
+            "Indicates if some sources of randomness in the experiments should be reduced by setting random_seed for numpy and torch. Default: ``False``.",
+        "type": "boolean"
+    }
+
+
+.. code-block:: JSON
+
+    {
+        "split_dataset": {
+            "control_randomness": true
```

Comment:
In our internal meeting, dated 18-07-2022 @naga-karthik suggested to perhaps have something along the lines `use_deterministic`. I guess `use_deterministic` could be slightly misleading as we aren't setting `torch.use_deterministic_algorithms(True)` as it would throw a `RuntimeError` due to the following operation in our codebase `F.interpolate()` (ref: [PyTorch docs](https://pytorch.org/docs/stable/generated/torch.use_deterministic_algorithms.html))